lib/AtomsIOPython/src/ase.jl

```diff
@@ -11,26 +11,48 @@ Supported formats:
   - ASE trajectory files
   - [XYZ](https://openbabel.org/wiki/XYZ) and [extxyz](https://github.com/libAtoms/extxyz#extended-xyz-specification-and-parsing-tools) files
 """
-struct AseParser <: AbstractParser end
+@kwdef struct AseParser <: AbstractParser 
+    read::Bool = true
```

Comment:
```suggestion
```
(see below)

Reply:
Yeah, this is an option. But then it won't be possible to set `read=false` when loading files. It would be useful to make it customizable.

Reply:
True, but I find it hard to imagine to have a use case where I wanted to read a file, but not want to open it first to read the magic bytes in order to determine the file type. Do you have one in mind?

Reply:
The `ase.io.formats.filetype` functions seems to have an unexpected behaviour when `read=True` and `guess=False` (which is the default behavior while reading a file in AtomsIO master branch). See for example,
```
>>> from ase.io import formats
>>> formats.filetype("test.xsf", read=True, guess=True)
'xsf'
>>> formats.filetype("test.xsf", read=False, guess=False)
'xsf'
>>> formats.filetype("test.xsf", read=False, guess=True)
'xsf'
>>> formats.filetype("test.xsf", read=True, guess=False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/rashid/Apps/mambaforge/envs/mlff/lib/python3.9/site-packages/ase/io/formats.py", line 954, in filetype
    raise UnknownFileTypeError('Could not guess file type')
ase.io.formats.UnknownFileTypeError: Could not guess file type
```
This occurs for other format also and this is the original reason I came up with issue. I think this is actually a bug in the implementation of `ase.io.formats.filetype` function in ASE. But I thought having it be customizable in AtomsIO would be better.

Reply:
Yikes, in case not yet done, please open an issue with ASE. Yes in this case let's have `guess = true` the default and make a comment in the code.

Reply:
Submitted an issue: https://gitlab.com/ase/ase/-/issues/1378